### ISSUE TITLE
Company Page review

### ISSUE THREAD

**Issue by cdolfi** [oss-aspen/8Knot#303]
For ease of visualization review and the need to merge the last PR lets use this Issue to discuss feedback on the new company page on 8knot. A hosted version can be found here: https://eightknot.osci.io/company and the code can be found in the visualization/company folder. Let me know if yall think of a better way of doing this discussion! Open to all feedback and ideas on graphs that might provide an additional perspective. We are also working on the descriptions as the differences between the graphs can be confusing #302

**Comment by oindrillac** [comment]
These charts are super amazing! Thanks for the great work @cdolfi 

Some UX feedback not ordered in any way:

1. It might be useful to have a checkbox to skip "gmail.com" and "noreply.github.com" domains to drill down better into company contributions. I'm thinking as a org stakeholder I'm interested more in the top contributing organizations
2. The date filter does not let us currently skip by years. Like if I want to select a date like 2022, I have to click 12*(2022-2005) times to reach there.
3. "Unique Contributor Email Domains" might be valuable to show what kinds of contributions comes under this? This can probably be added in the Above Graph section? something like "this includes commits, PRs etc etc"
4. Spell error? "initity" in Company Associated Activity about graph. Also the text "The methodology behind this is to take each associated email to someones github account and link the contributions to each as it is unknown which initity the actvity was done for." was not very clear to me. Can you please explain or can this be rephrased? How is "Company Associated Activity" different from "Unique Contributor Email Domains"?
5. Contributions Required: is such a cool feature! ++

**Comment by cdolfi** [comment]
@oindrillac Thank you for all of this! I actually had 1 before but was a double to being able to click on the name in the legend to remove it. We need to figure out how to make sure users know that functionality is there. 2 unfortunately is a dash object that we can change, luckily there is the ability to type in the date. 3 and 4 we will definitely take into account with #302

**Comment by oindrillac** [comment]
Oh nice, was not aware of the double click feature until you mentioned, that is useful. Maybe hovering on the legends can give a text like double click to isolate (I saw that text pop up on the top right hand side of the window once I double clicked and then it dissapeared) <img width="202" alt="Screenshot 2023-05-05 at 12 49 05 AM" src="https://user-images.githubusercontent.com/32435206/236306878-06e21cc7-f34d-491f-8408-7a81f417421c.png">

> type in the date
Oh HAHA I did not realize we could modify the text. That is intuitive, just my bad.
